src/client/public/css/components/review.css

```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Yeseva+One&display=swap");
```

Comment:
import it the same way as inter, we can use it not only in the review component in the future